docs/micro-frontends-in-open-edx.rst

```diff
@@ -0,0 +1,95 @@
+#######################################
+Micro-frontend applications in Open edX
+#######################################
+
+The purpose of this document is to provide an overview of how micro-frontend applications (MFEs) are developed, configured, and deployed with the Open edX ecosystem. Open edX MFEs are small React applications that can be built and deployed independently.
+
+MFEs are static site generators. They are supplied configuration, then run to build HTML, CSS, and Javascript. The static assets are then ready then be moved to a location and served to the user.
+
+All MFEs offer the following commands:
+
+.. code-block::
+
+  npm install (install dependencies)
+  npm start (development server)
+  npm run lint
+  npm run test
+  npm run build (build static output)
+
+Configuration with Environment Variables
+----------------------------------------
+
+Open edX MFEs expect a number of process environment variables (Accessible in node via `process.env`) in order to properly configure authentication, routing to other parts of the system, etc. Each MFE has `.env`, `.env.development`, and `.env.test` files that are consumed by the MFE. The production .env file contains only null values – the build process should supply them via the command line prior to npm run build.
+
+Example `.env.development`:
+
+.. code-block:: bash
+
+  NODE_ENV='development'
+  PORT=1995
+  ACCESS_TOKEN_COOKIE_NAME='edx-jwt-cookie-header-payload'
+  BASE_URL='localhost:1995'
+  CREDENTIALS_BASE_URL='http://localhost:18150'
+  CSRF_COOKIE_NAME='csrftoken'
```

Comment:
CSRF_COOKIE_NAME is not actually used - I've been removing it from the MFEs.  Should nix it.